docs/howto/sqlite-json1.txt

```diff
@@ -0,0 +1,87 @@
+=======================================
+How to enable JSON1 extension on SQLite
+=======================================
```

Comment:
I think this can live as a (long) section under `ref/databases.txt` under "SQLite Notes"